### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by Isarstyle** [BelkaLab/react-yearly-calendar#21]
I love the Calendar, is it still maintained? 

´Found an Issue while selecting a Range the selectingRange prop is not defined. Is it even needed?
`var newRangeStart = nextProps.selectingRange[0].month();`

```
Uncaught TypeError: Cannot read property '0' of undefined
    at Month.shouldComponentUpdate (Month.js:89)
    at eval (ReactCompositeComponent.js:622)
    at measureLifeCyclePerf (ReactCompositeComponent.js:73)
    at ReactCompositeComponentWrapper.updateComponent (ReactCompositeComponent.js:621)
    at ReactCompositeComponentWrapper.receiveComponent (ReactCompositeComponent.js:544)
    at Object.receiveComponent (ReactReconciler.js:122)
    at Object.updateChildren (ReactChildReconciler.js:107)
    at ReactDOMComponent._reconcilerUpdateChildren (ReactMultiChild.js:204)
    at ReactDOMComponent._updateChildren (ReactMultiChild.js:308)
    at ReactDOMComponent.updateChildren (ReactMultiChild.js:295)

```
Thank you.

**Comment by GiovanniFrigo** [comment]
Hello @Isarstyle, yes, the calendar is still maintained. I'm doing some work on it right now.
I will look at your issues ASAP!

Have a nice weekend!

**Comment by BIWhitfield** [comment]
Loving this calendar too but +1 for this error. Any progress?

**Comment by GiovanniFrigo** [comment]
Fixed this issue in version 1.2.1
